health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/SelectingBoundariesDuplicate.js

```
@@ -77,52 +89,62 @@ const SelectingBoundariesDuplicate = ({ onSelect, formData, ...props }) => {
+                handleBoundaryChange(value);
+              }}
+            ></Wrapper>
+          </Card>
+          <InfoCard
+            populators={{
+              name: "infocard",
+            }}
+            variant="default"
+            style={{ margin: "0rem", maxWidth: "100%" }}
+            additionalElements={[
+              <span style={{ color: "#505A5F" }}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)